sunpy/timeseries/sources/eve.py

```diff
@@ -78,15 +80,18 @@ def plot(self, axes=None, **kwargs):
         colors = ('tab:blue', 'tab:orange', 'tab:green', 'tab:red', 'tab:purple',
                   'tab:brown', 'tab:pink', 'tab:gray', 'tab:olive', 'tab:cyan')
         predefined_axes = False
+        if column is None:
+            column = self.to_dataframe().columns
         if isinstance(axes, np.ndarray):
             predefined_axes = True
         elif axes is None:
-            axes = self.to_dataframe().plot(subplots=True, sharex=True, **kwargs)
+            axes = self.to_dataframe()[column].plot(subplots=True, sharex=True, **kwargs)
         for i, ax in enumerate(axes):
             if predefined_axes:
-                ax.plot(self._data[self._data.columns[i]],
-                        color=colors[i % len(colors)],
-                        label=self._data.columns[i])
+                if i in column:
+                    ax.plot(self._data[self._data.columns[i]],
+                            color=colors[i % len(colors)],
+                            label=self._data.columns[i])
```

Comment:
How does this work?

Reply:
Sorry, my bad I messed up there. I am on it...